### ISSUE TITLE
Py 3 does not have a long type

### ISSUE THREAD

**Issue by oberstet** [cocagne/txdbus#67]
split out from #11 

```
(cpy361_7) oberstet@thinkpad-t430s:~/scm/3rdparty/txdbus$ flake8 --select=F821 --show-source .
./txdbus/test/test_marshal.py:39:20: F821 undefined name 'long'
            self.t(long(1),'x')
                   ^
```

here:

```
(cpy361_7) oberstet@thinkpad-t430s:~/scm/crossbario/crossbario-organize/fibu$ python
Python 3.6.1 (default, Apr  2 2017, 18:19:20) 
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import six
>>> six.integer_types
(<class 'int'>,)
>>> 
(cpy361_7) oberstet@thinkpad-t430s:~/scm/crossbario/crossbario-organize/fibu$ python2
Python 2.7.12 (default, Nov 19 2016, 06:48:10) 
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import six
>>> six.integer_types
(<type 'int'>, <type 'long'>)
>>> 
KeyboardInterrupt
>>> 
```

**Comment by exvito** [comment]
Even though its worded differently, I think this duplicates #61 which has a proposed fix in PR #65. Do you agree?

**Comment by oberstet** [comment]
ah, right. dup
